src/plotting/plot_tools.py

```diff
@@ -207,3 +207,32 @@ def load_plots(
         f" {time.time()-start_time} seconds"
     )
     return changed, new_provers, failed_to_open_filenames, no_key_filenames
+
+
+def find_duplicate_plot_IDs(all_filenames: List[Path] = []) -> None:
+    plot_IDs_set = set()
+    duplicate_plot_IDs = set()
+    all_filenames_str: List[str] = []
+
+    for filename in all_filenames:
+        filename_str: str = str(filename)
```

Comment:
Actually a better way to do this, that does not require filenames to have a specific format, is this:
plot_id =  DiskProver(str(filename)).get_id()

Because the plot id is written into the file.